pkgs/tools/misc/toybox/default.nix

```diff
@@ -20,6 +20,9 @@ stdenv.mkDerivation rec {
   depsBuildBuild = [ buildPackages.stdenv.cc ]; # needed for cross
```

Comment:
This is what causes the behaviour in #210923, actually. Should be good when using `stdenv.cc` on x86_64-darwin.

Reply:
Oh man, nice catch!